cli/README.md

```diff
@@ -4,12 +4,18 @@
 
 ## Example usage:
 
+1. In a terminal, Run following commands to use rill cli:
+```
+make cli
+```
+
 ```sh
-rill --help
-rill version
-rill init [name]
-rill start [path]
-rill import-source [path]
-rill drop-source [path]
-rill init-example
+./rill --help
+./rill version
+./rill docs
+./rill init [name]
+./rill start [path]
+./rill import-source [path]
+./rill drop-source [path]
+./rill init-example
 ```
```

Comment:
1. Don't need to list all the commands here, just `./rill help` should be enough
2. Make it clear that this is in development

Maybe just:

> In development, run the following to build and run the CLI:
> ```
> make cli
> ./rill help
> ```

Or something like that